code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -1,36 +1,139 @@
+	if(I.force)
+		..()
+		playsound(src, I.hitsound, 50, 1)
+		health = Clamp(health - I.force, 0, initial(health))
+		if(health <= 0)
+			destroy_structure()
+		return
+	..()
 
 /obj/structure/cult/talisman
-	name = "altar"
-	desc = "A bloodstained altar dedicated to Nar-Sie"
+	name = "sacrificial altar"
+	desc = "An altar made of tough wood and draped with an ornamental, bloodstained cloth."
 	icon_state = "talismanaltar"
-
+	health = 150 //Sturdy
+	death_message = "<span class='warning'>The altar breaks into splinters, releasing a cascade of spirits into the air!</span>"
+	death_sound = 'sound/effects/altar_break.ogg'
 
 /obj/structure/cult/forge
-	name = "daemon forge"
-	desc = "A forge used in crafting the unholy weapons used by the armies of Nar-Sie"
+	name = "runed forge"
+	desc = "A combination furnace and anvil. It glows with the heat of the lava flowing through its channel."
 	icon_state = "forge"
+	luminosity = 2
+	health = 300 //Made of metal
+	death_message = "<span class='warning'>The forge falls apart, its lava cooling and winking away!</span>"
+	death_sound = 'sound/effects/forge_destroy.ogg'
 
 /obj/structure/cult/pylon
-	name = "pylon"
-	desc = "A floating crystal that hums with an unearthly energy"
+	name = "energy pylon"
+	desc = "A hovering red crystal that thrums with energy and light. Kept aloft by two metal prongs."
 	icon_state = "pylon"
 	luminosity = 5
+	health = 50 //Very fragile
+	death_message = "<span class='warning'>The pylon's crystal vibrates and glows fiercely before violently shattering!</span>"
+	death_sound = 'sound/effects/Glassbr2.ogg'
+
+/obj/structure/cult/pylon/destroy_structure()
+	for(var/mob/living/M in range(5, src))
+		if(!issilicon(M))
+			M.visible_message("<span class='warning'>Deadly shards of red crystal impact [M]!</span>", \
+							"<span class='userdanger'>Deadly red crystal shards fly into you!</span>")
+			M.adjustBruteLoss(rand(5,10))
+		else
+			M.visible_message("<span class='warning'>Red crystal shards bounce off of [M]'s casing!</span>", \
+							"<span class='userdanger'>Red crystal shards bounce off of your casing!</span>")
+	..()
 
 /obj/structure/cult/tome
-	name = "desk"
-	desc = "A desk covered in arcane manuscripts and tomes in unknown languages. Looking at the text makes your skin crawl"
+	name = "research desk"
+	desc = "A writing desk covered in strange volumes written in an unknown tongue."
 	icon_state = "tomealtar"
 	luminosity = 1
+	health = 125 //Slightly sturdy
+	death_message = "<span class='warning'>The desk breaks apart, its books falling to the floor.</span>"
+	death_sound = 'sound/effects/wood_break.ogg'
 
 /obj/effect/gateway
 	name = "gateway"
-	desc = "You're pretty sure that abyss is staring back"
+	desc = "You're pretty sure that abyss is staring back."
 	icon = 'icons/obj/cult.dmi'
 	icon_state = "hole"
 	density = 1
 	unacidable = 1
 	anchored = 1
+
+////////////////
+// CULT TURFS //
+////////////////
+
+/turf/simulated/floor/plasteel/cult

```

Comment:
You need to do this for `/turf/simulated/floor/engine/cult` as well, or preferably remove one of those paths.